eclair-core/src/test/scala/fr/acinq/eclair/payment/PaymentLifecycleSpec.scala

```diff
@@ -713,7 +713,7 @@ class PaymentLifecycleSpec extends BaseRouterSpec {
     import payFixture._
     import cfg._
 
-    val request = SendPayment(sender.ref, d, Onion.createSinglePartPayload(defaultAmountMsat, defaultExpiry, defaultInvoice.paymentSecret.get), 3)
```

Comment:
We need to add tests to this file to verify that we're not retrying `SendPaymentToRoute`.